project/docs/macros.py

```diff
@@ -31,9 +31,9 @@ def get_credits_data() -> dict:
     dependencies = direct_dependencies | indirect_dependencies
 
     packages = {}
-    for pkg in search_packages_info(dependencies):
-        pkg = {_: pkg[_] for _ in ("name", "home-page")}
-        packages[pkg["name"].lower()] = pkg
+    for pkg in search_packages_info(list(dependencies)):
+        pkg = {_: pkg[_] for _ in ("name", "home-page")}  # type: ignore
+        packages[pkg["name"].lower()] = pkg  # type: ignore
```

Comment:
Not ideal but i think unless this is an easy fix should not be causing new users using this to start off with failing types when they are most likely not going to be experts in this area.